package.json

```diff
@@ -1,123 +1,36 @@
 {
   "name": "preact-realworld-example-app",
   "version": "0.1.2",
-  "description": "> ### [Preact X](https://github.com/developit/preact) codebase containing real world examples (typescript, preact-hooks, preact-router, fetch, etc) that adheres to the [RealWorld](https://github.com/gothinkster/realworld) spec and API.",
-  "keywords": [
-    "realworld",
-    "preact",
-    "typescript"
-  ],
-  "author": "mutoe <mutoe@foxmail.com>",
-  "license": "MIT",
-  "homepage": "https://github.com/mutoe/preact-realworld-example-app#readme",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/mutoe/preact-realworld-example-app.git"
-  },
-  "bugs": {
-    "url": "https://github.com/mutoe/preact-realworld-example-app/issues"
-  },
-  "main": "src/index.tsx",
```

Comment:
Just so you know for the future, all of these are only relevant for packages. They have 0 reason to exist on an application. It's meta data for package repositories like https://npmjs.com